docs/user-guide/release-notes/2.0.x.rst

```diff
@@ -1,6 +1,19 @@
 2.0 Release Notes
 =================
 
+2.0.1
```

Comment:
2.0.1 is already releasing, so this would be for 2.0.2. Redmine is a little out of sync for the docker project, so I'll be getting that cleaned up tomorrow.

I the meantime, here's a PR for the 2.0.1 release notes so this can get bumped to 2.0.2:
https://github.com/pulp/pulp_docker/pull/150